packages/landing/static/json-schema/basemaps-extension/1.0/schema.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://basemaps.linz.govt.nz/json-schema/basemaps-extension/1.0/schema.json#",
+  "title": "LINZ Basemaps Extension",
+  "description": "STAC LINZ Basemaps extension to a SATC item and STAC collection.",
+  "oneOf": [
+    {
+      "allOf": [
+        {"$ref": "https://schemas.stacspec.org/v1.0.0-beta.2/item-spec/json-schema/item.json"},
+        {"$ref": "#/definitions/linz-common"},
+        {"$ref": "#/definitions/linz-item"}
+      ],
+    },
+    {
+      "allOf": [
+        {"$ref": "https://schemas.stacspec.org/v1.0.0-beta.2/collection-spec/json-schema/collection.json"},
+        {"$ref": "#/definitions/linz-common"},
+        {"$ref": "#/definitions/linz-collection"},
```

Comment:
we cant use `linz-common` maybe `linz-basemaps-common` ?

Reply:
It is just a local ref; we can use whatever we want

Reply:
I changed it to just "common", "collection" and "item"